src/main/UseCases/AddInputBoundary.java

```diff
@@ -0,0 +1,12 @@
+package UseCases;
+
+public interface AddInputBoundary {
```

Comment:
Based on clean architecture, this interface is created for not violating dependency, and can be implemented by AddUseCase for transporting data purposes.